index.js

```diff
@@ -0,0 +1,70 @@
+
+function validateLineup(items) {
+  let toggle = true
+  //  separates salaries into new array
+  const salaries = items.map((player) => player.salary)
+  //  separates teams into new array
+  const teams = items.map((player) => player.teamId)
+  //  separates games into new array
+  const games = items.map((player) => player.gameId)
+  //  separates positions into new array
+  const positions = items.map((player) => player.position)
+
+  //  determines if position allocation limits have been met
+  if (positions.filter((position) => position === 'OF').length < 3 ||
+    positions.filter((position) => position === 'OF').length > 3) { return false }
+  if (positions.filter((position) => position === 'P').length < 1 ||
+    positions.filter((position) => position === 'P').length > 1) { return false }
+  if (positions.filter((position) => position === 'C').length < 1 ||
+    positions.filter((position) => position === 'C').length > 1) { return false }
+  if (positions.filter((position) => position === '1B').length < 1 ||
+    positions.filter((position) => position === '1B').length > 1) { return false }
+  if (positions.filter((position) => position === '2B').length < 1 ||
+    positions.filter((position) => position === '2B').length > 1) { return false }
+  if (positions.filter((position) => position === '3B').length < 1 ||
+    positions.filter((position) => position === '3B').length > 1) { return false }
+  if (positions.filter((position) => position === 'SS').length < 1 ||
+    positions.filter((position) => position === 'SS').length > 1) { return false }
+
+  // callback function for reduce method (not sure if needed???)
+  function sum(acc, n) {
+    return acc + n
+  }
+
+  //  totals and checks if salaries exceed limit
```

Comment:
I added salaries manually, then consoled to make sure it was getting the correct total. Then I increase one of the salaries to push over the limit to see if it would return false and it was successful. Is it the callback function that's setup wrong? I don't see it.